Podfile

```diff
@@ -1,2 +1,7 @@
 platform :ios, '7.1'
-pod 'measurement_kit'
+target 'NetProbe' do
+	pod 'measurement_kit', '~> 0.1.2.1'
```

Comment:
@lorenzoPrimi, quick tips to have it compile again (from my previous take at it):

1) try with platform = 9.0: the library on cocoapods was compiled with 9.0 not 7.1; we will set 7.1 later

2) use '0.1.2' not '0.1.2.1'

With those two settings it worked for me. Perhaps it would work again :-D